### PR TITLE
Add Fang's Taipei Recommendations File

### LINKED ISSUE #8: Create a file for Taipei, Taiwan Recommendations
# Taipei, Taiwan

- [ ] Create a branch
- [ ] Add a commit
- [ ] Open a Pull Request
- [ ] Have a discussion
- [ ] Seek approval
- [ ] Merge Pull Request

### PR DESCRIPTION
**Pull request by FLIN1025**
closes #8

### CODE REVIEW

File: fang1025-Taipei.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Taipei, Taiwan
+
+### Places to Eat
+
```
Comment (davidjgithubtraining) [inline fang1025-Taipei.md]: any recommendations?

**Comment by davidjgithubtraining** [comment]
🥇

File: fang1025-Taipei.md (hunk 1 above)
Comment (arde5730) [reply fang1025-Taipei.md]: @davidjgithubtraining 
Google it???

Comment (FLIN1025) [reply fang1025-Taipei.md]: @davidjgithubtraining @arde5730 places added. forgot to push.

**Review by davidjgithubtraining — APPROVED** [review]
lot's of recommendations: :-)

**Pull request merged**